### PR TITLE
Creates a helper method to set some better MySQL global values

### PR DESCRIPTION
**Pull request by jaredcobb**

### CODE REVIEW

File: inc/database/class-mysql-processor.php [hunk 1]
```diff
@@ -37,9 +37,30 @@ public function get_connection() {
...
+		$this->set_mysql_global_defaults( $connection );
 		return $connection;
 	}
 
+	/**
+	 * Set any global MySQL default settings and customizations
+	 * for this session.
+	 *
+	 * @param    PDO   $connection Current PDO connection
+	 *
```
Comment (timatron) [inline inc/database/class-mysql-processor.php]: cut this extra line? standard doctoc?

File: inc/database/class-mysql-processor.php [hunk 2]
```diff
@@ -37,9 +37,30 @@ public function get_connection() {
...
+	 * @return   void
+	 */
+	public function set_mysql_global_defaults( $connection ) {
+		$defaults = [
+			'group_concat_max_len' => 50000,
+		];
+
+		foreach ( $defaults as $key => $value ) {
+			$stmt = $connection->prepare("SET SESSION {$key} = {$value}");
+			$stmt->execute();
```
Comment (timatron) [inline inc/database/class-mysql-processor.php]: check for no error afterwards?

**Comment by timatron** [comment]
🦉

File: inc/database/class-mysql-processor.php (hunk 1 above)
Comment (jaredcobb) [reply inc/database/class-mysql-processor.php]: hmm, i always thought `@return` in it's own grouping (space between params and return) http://docs.phpdoc.org/getting-started/your-first-set-of-documentation.html#example

File: inc/database/class-mysql-processor.php (hunk 2 above)
Comment (jaredcobb) [reply inc/database/class-mysql-processor.php]: sounds good

**Pull request merged**